### PR TITLE
Add New Church Request

### PR DESCRIPTION
**Pull request by Ashar67**
Members without existing church affiliation should be able to send a request to the admins for adding a new church to the system to invite other members to, of which they will become the leader. Upon admin approval, the church will be activated.

### CODE REVIEW

File: internal/storage/dal/church.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package dal
+
+import (
+	"github.com/google/uuid"
+	"github.com/jsfan/hello-neighbour-api/internal/storage/models"
+	"github.com/jsfan/hello-neighbour-api/pkg"
+)
+
+func (dalInstance *DAL) InsertChurch(churchIn *pkg.ChurchIn) error {
```
Comment (jsfan) [inline internal/storage/dal/church.go:9]: I usually use `QueryContext` an then put a `RETURNING id` at the end of the query. That way, you get the DB id right with the insert.

**Review by jsfan — APPROVED** [review]
@Ashar67 I didn't go over it too thoroughly but it looks good to me. I saw that you have a few TODOs for tests left. So, I'll leave it to you to merge or add those.

**Review by jsfan — APPROVED** [review]

**Comment by Ashar67** [comment]
Not sure how to implement the tests on the DAL level. Unless it's the same format as what we did on the storage layer?

**Comment by jsfan** [comment]
> Not sure how to implement the tests on the DAL level. Unless it's the same format as what we did on the storage layer?

Don't worry about it for now. The tests will need a running database server but they probably aren't the highest value tests we could have, anyway.

**Review by jsfan — APPROVED** [review]

**Pull request merged**